scripts/hudson/narayana.sh

```diff
@@ -60,13 +60,15 @@ function init_test_options {
     is_ibm
     ISIBM=$?
     [ $NARAYANA_CURRENT_VERSION ] || NARAYANA_CURRENT_VERSION="5.8.2.Final-SNAPSHOT"
+    [ $MICROPROFILE_LRA_VERSION ] || MICROPROFILE_LRA_VERSION="1.0-SNAPSHOT"
+    [ $MICROPROFILE_LRA_BRANCH ] || MICROPROFILE_LRA_BRANCH="microprofile-lra"
     [ $CODE_COVERAGE ] || CODE_COVERAGE=0
     [ x"$CODE_COVERAGE_ARGS" != "x" ] || CODE_COVERAGE_ARGS=""
     [ $ARQ_PROF ] || ARQ_PROF=arq	# IPv4 arquillian profile
     [ $IBM_ORB ] || IBM_ORB=0
 
     PULL_DESCRIPTION=$(get_pull_description)
-    
```

Comment:
formatting only change?

Reply:
I think I need to disable one of my exuberant a vim plugins.

Reply:
:)